model-optimizer/unit_tests/mo/frontend_ngraph_test_actual.py

```diff
@@ -227,3 +254,19 @@ def test_error_batch(self, mock_argparse):
         assert stat.get_partial_shape == 1
         # verify that 'set_element_type' was not called
         assert stat.set_partial_shape == 0
+
+    @mock_needed
+    @patch('argparse.ArgumentParser.parse_args',
+           return_value=replaceArgsHelper(input_model='abc.qwerty'))
+    def test_error_input_model_no_framework(self, mock_argparse):
+        # Framework is not specified and 'abc.qwerty' is not supported
+        # so MO shall not convert anything and produce specified error
+        with self.assertLogs() as logger:
+            main(argparse.ArgumentParser(), fem, None)
+
+        stat = get_frontend_statistic()
+
+        assert [s for s in logger.output if 'can not be deduced' in s]
+
+        # verify that 'supported' was called
+        assert stat.supported == 1
```

Comment:
A generic comment for all unit tests. Use `self.assertEqual(stat.supported, 1)` and similar functions to do asserts. If the output is wrong then you will get detailed message what were the parameters for the function call.